lenny/models/book.py

```diff
@@ -0,0 +1,18 @@
+from sqlmodel import SQLModel, Field
+from typing import Optional
+from enum import Enum
+
+
+class BookStatus(str, Enum):
+    AVAILABLE = "available"
+    BORROWED = "borrowed"
+
+
+class Book(SQLModel, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
+    title: str
+    author: str
+    isbn: str
```

Comment:
We won't need book metadata, we can't guarantee the item will be a book, and many books do not have ISBN, especially those pre-1975